### PR TITLE
New 'center' parameter to whether to center the tracking component.

### PR DESCRIPTION
**Pull request by olivierauverlot**

### CODE REVIEW

File: examples/official-site/sqlpage/migrations/32_tracking_update.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    )
+VALUES (
+        'tracking',
+        'center',
+        'Whether to center the component.',
+        'BOOLEAN',
+        TRUE,
+        TRUE
+    );
+
```
Comment (lovasoa) [inline examples/official-site/sqlpage/migrations/32_tracking_update.sql:17]: Can we keep that in the existing tracking component documentation file ?

The database behind sql.ophir.dev is in-memory anyway and migrations are reapplied on every deploy.

**Review by lovasoa — CHANGES_REQUESTED** [review]
I'd like to make the docs easier to edit by keeping them in one file per component. 

The change itself is good

File: examples/official-site/sqlpage/migrations/32_tracking_update.sql (hunk 1 above)
Comment (olivierauverlot) [reply examples/official-site/sqlpage/migrations/32_tracking_update.sql:17]: ok, I will merge the two files :)

**Comment by olivierauverlot** [comment]
The files are merged

**Review by lovasoa — APPROVED** [review]

**Comment by lovasoa** [comment]
Thanks @olivierauverlot !

**Pull request merged**